### PR TITLE
Rename detector_mode by detector_binning_mode

### PR DESCRIPTION
**Pull request by jordiandreu**
Rename *detector_mode* by *detector_binning_mode* after the addition of the *detector_roi_mode* parameter.

### CODE REVIEW

File: HardwareObjects/mockup/DetectorMockup.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def init(self):
         self.actual_frame_rate = 50
         self.roi_modes_list = ("0", "C2", "C16")
         self.roi_mode = 0
```
Comment (IvarsKarpics) [inline HardwareObjects/mockup/DetectorMockup.py]: this should be self._roi_mode

**Comment by IvarsKarpics** [comment]
@jordiandreu  Could you please update the branch with the latest changes from the master?

**Comment by rhfogh** [comment]
@IvarsKarpics @jordiandreu 
I have done the updating.

**Comment by rhfogh** [comment]
continuous-integration/travis-ci/pr build failed. yet clicking on details I do not find anything telling me what failed or why or caht could be done about it.

What am I missing?

I'd merge this after looking at the code, but better not as it stands.

File: HardwareObjects/mockup/DetectorMockup.py (hunk 1 above)
Comment (jordiandreu) [reply HardwareObjects/mockup/DetectorMockup.py]: Indeed, also self._roi_mode_list, self._temperature, etc...Done

**Comment by IvarsKarpics** [comment]
Hi @rhfogh . In this case the detector pytest is failing. Clearly it says that the values is not within the limits. 
For everyone is it worth to check manually if the pytest runs through:
`python -m pytest`

**Comment by rhfogh** [comment]
@IvarsKarpics  Is there anywhere on github where is shows what actually went wrong with those tests?

**Comment by marcus-oscarsson** [comment]
Thanks, great !

**Comment by marcus-oscarsson** [comment]
It looks from the tests that detector_distance is not configured !?

**Comment by marcus-oscarsson** [comment]
The PR is otherwise looks good to me !

**Comment by marcus-oscarsson** [comment]
I can merge this as it is if you make another PR for the test, or you can fix the test in this PR. Because it was working before right ?

**Comment by jordiandreu** [comment]
I will fix the test and upload the changes, I let you know!

**Comment by marcus-oscarsson** [comment]
Actually Ill merge this and we will take a look at the test in another PR, I wont hold this up any longer.

**Comment by marcus-oscarsson** [comment]
Ohh, sorry noticed your comment to late ... :)

**Comment by jordiandreu** [comment]
No  problem, I will upload the changes in a new PR.

**Pull request merged**